deployments/helm-chart/templates/controller-daemonset.yaml

```
@@ -111,16 +111,23 @@ spec:
 {{- end }}
         securityContext:
           allowPrivilegeEscalation: true
+          readOnlyRootFilesystem: true
```

```suggestion
          readOnlyRootFilesystem: {{ .Values.controller.readOnlyRootFilesystem }}
```
This will require adding `readOnlyRootFilesystem: false` to the `controller` section in `deployments/helm-char/values.yaml`

These same changes can be applied to `controller-deployment.yaml`